app/views/root/local_transaction.html.erb

```diff
@@ -65,7 +55,7 @@
         "userAlerts:LocalTransaction",
         "postcodeErrorShown:<%= @location_error.postcode_error %>",
         {
-          label: "<%= t(@location_error.message, @location_error.message_args) %>"
+          label: "Search the <%= @location_error.message_args[:local_authority_name] %> website for this service."
         }
```

Comment:
I'm wondering if it will alway send the "Search the..." label even when it's not appropriate such as `invalidPostcodeFormat`? What happens when `@location_error.message_args` is `nil`?